src/libraries/System.Numerics.Tensors/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,214 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please add the following lines to the suppression file (change the assembly names): https://github.com/dotnet/runtime/blob/6d76479447b67e119173fa4cd3acb3a2e1887cd3/src/libraries/Microsoft.Extensions.Caching.Abstractions/src/CompatibilitySuppressions.xml#L4-L14

This will make sure that APICompat doesn't emit errors for TFMs that we intentionally dropped in the current package (net6.0 and net7.0). We added these lines to all our packable projects in main as we currently don't have mechanism to suppress those TFMs globally. That work is tracked via https://github.com/dotnet/sdk/issues/22901

You can then remove the following entries:
- left: `lib/net6.0 or `lib/net7.0` compared against
- right: `lib/netstandard2.0`

cc @ericstj as there is quite a lot of room for improvements

Reply:
@ViktorHofer if you see room for improvements file issues for those - I'm not certain what in particular you were referring to here.

Reply:
https://github.com/dotnet/sdk/issues/22901 which Santiago filed two years ago. Especially now, as we decided to suppress net6.0 and net7.0 compatibility errors in all packable projects.